iOS/DuckDuckGo/TabSwitcherViewController.swift

```diff
@@ -122,38 +126,98 @@ class TabSwitcherViewController: UIViewController {
 
     let barsHandler = TabSwitcherBarsStateHandler()
     private var tabObserverCancellable: AnyCancellable?
+    private let appSettings: AppSettings
 
     required init?(coder: NSCoder,
                    bookmarksDatabase: CoreDataDatabase,
                    syncService: DDGSyncing,
                    featureFlagger: FeatureFlagger,
                    favicons: Favicons = Favicons.shared,
                    tabManager: TabManager,
-                   aiChatSettings: AIChatSettingsProvider) {
+                   aiChatSettings: AIChatSettingsProvider,
+                   appSettings: AppSettings) {
         self.bookmarksDatabase = bookmarksDatabase
         self.syncService = syncService
         self.featureFlagger = featureFlagger
         self.favicons = favicons
         self.tabManager = tabManager
         self.aiChatSettings = aiChatSettings
+        self.appSettings = appSettings
         super.init(coder: coder)
     }
 
     required init?(coder: NSCoder) {
         fatalError("Not implemented")
     }
 
-    fileprivate func createTopBar() {
+    fileprivate func createTitleBar() {
         let appearance = UINavigationBarAppearance()
         appearance.configureWithTransparentBackground()
-        topBarView.standardAppearance = appearance
-        topBarView.scrollEdgeAppearance = appearance
+        titleBarView.standardAppearance = appearance
+        titleBarView.scrollEdgeAppearance = appearance
+    }
+
+    private func activateLayoutConstraintsBasedOnBarPosition() {
+        let isBottomBar = appSettings.currentAddressBarPosition.isBottom
+
+        // Potentially for these 3 we could do thing better for 'normal' on iPad
+        let topOffset = -6.0
+        let bottomOffset = 8.0
+        let navHPadding = isExperimentalThemingEnabled ? 10.0 : 0.0
+
+        // The constants here are to force the ai button to align between the tab switcher and this view
+        NSLayoutConstraint.activate([
```

Comment:
One alternate approach here is to save these constraints in a property which we can then deactivate later, e.g. `dynamicConstraints = ...`